cardano-cli/CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog for cardano-cli
 
+## 10.7.0.0
+
+- Bump cardano-api to include bug fixes for:
+  -  Silently dropping simple scripts in transaction construction
+  -  Not selecting the highest protocol version in a given era. This resulted in erroneous plutus script decoding failures.
+  (compatible, bugfix)
```

Comment:
```suggestion

  (compatible, bugfix)
```
Without an empty line before the tags it assigns them to the sub-bullet (see the rendering of the Markdown)